packages/admin/admin-date-time/src/DatePickerNavigation.tsx

```diff
@@ -5,7 +5,7 @@ import { css, Theme, useThemeProps } from "@mui/material/styles";
 import { useRef, useState } from "react";
 import { useIntl } from "react-intl";
 
-export interface DatePickerNavigationProps
+interface DatePickerNavigationProps
```

Comment:
This component isn't added to index.ts, so removing should be fine.

Reply:
done